notebooks/README.md

```diff
@@ -1,10 +1,77 @@
+[<img src="https://img.shields.io/badge/-Setup Studio Lab Environment-gray.svg">](#amazon-sagemaker-studio-lab) [<img src="https://img.shields.io/badge/-Setup Colab Environment-gray.svg">](#google-colab)
 
+1. Mortgage_example.ipynb
 
-- Auto dataset from [here](https://drive.google.com/file/d/1jxySYJ9e32hI8PQ5QPr9_xrsu37N5fOM/view?usp=sharing)
+    [![Open In Studio Lab](https://studiolab.sagemaker.aws/studiolab.svg)](https://studiolab.sagemaker.aws/import/github/rapidsai/cuxfilter/blob/branch-22.02/notebooks/Mortgage_example.ipynb) [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/rapidsai/cuxfilter/blob/branch-22.02/notebooks/Mortgage_example.ipynb) 
+
+2. NYC_taxi_example.ipynb
+
+    [![Open In Studio Lab](https://studiolab.sagemaker.aws/studiolab.svg)](https://studiolab.sagemaker.aws/import/github/rapidsai/cuxfilter/blob/branch-22.02/notebooks/NYC_taxi_example.ipynb) [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/rapidsai/cuxfilter/blob/branch-22.02/notebooks/NYC_taxi_example.ipynb)
+
+3. auto_accidents_example.ipynb
+
+    [![Open In Studio Lab](https://studiolab.sagemaker.aws/studiolab.svg)](https://studiolab.sagemaker.aws/import/github/rapidsai/cuxfilter/blob/branch-22.02/notebooks/auto_accidents_example.ipynb) [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/rapidsai/cuxfilter/blob/branch-22.02/notebooks/auto_accidents_example.ipynb)
+
+4. graphs.ipynb
+
+    [![Open In Studio Lab](https://studiolab.sagemaker.aws/studiolab.svg)](https://studiolab.sagemaker.aws/import/github/rapidsai/cuxfilter/blob/branch-22.02/notebooks/graphs.ipynb) 
+    [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/rapidsai/cuxfilter/blob/branch-22.02/notebooks/graphs.ipynb)
+</br>
+
+
+## Setup Remote Environments
+### Amazon Sagemaker Studio Lab
+
+[Amazon SageMaker Studio Lab](https://studiolab.sagemaker.aws/faq) is a free online web application for learning and experimenting with data science and machine learning using Jupyter notebooks and is free to use.
+
+Once you have registered with your email address, simply sign in to your account, start a CPU or GPU runtime, and open your project - all in your browser.
+
+To setup a rapids environment in studio lab(you only need to do this the first time, since studio lab has 15GB of persistent storage across sessions), open a new terminal:
+
+```bash
+conda install ipykernel
+
+# for stable rapids version
+conda install -c rapidsai -c nvidia -c numba -c conda-forge \
+    cuxfilter python=3.7 cudatoolkit=11.5
+
+# for nightly rapids version
+conda install -c rapidsai-nightly -c nvidia -c numba -c conda-forge \
+    cuxfilter python=3.7 cudatoolkit=11.5
+```
```

Comment:
I'd make a note that these are sample conda install snippets, but they should go to the rapids get started page for the latest (w/ link). Then we can update them less periodically.

Reply:
added a note as mentioned. Thanks :)